docker-compose.yml

```diff
@@ -114,6 +120,8 @@ services:
     hostname: auth
     networks:
       placeos:
+    healthcheck:
+      test:  wget --no-verbose -q --spider http://0.0.0.0:8080/auth/authority?health=true
```

Comment:
https://github.com/PlaceOS/auth/pull/31

Reply:
Maybe we should keep it for a few versions as older images won’t have the explicit healthcheck.

Let’s say remove in 2 versions?